site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-list.vm

```diff
@@ -270,12 +270,16 @@
 								#end
 							</td>
 							<td headers="title">
+								#if (!$service.allowAccessSite($site.Id))
+                            		<label style="opacity:0.2">$validator.escapeHtml($site.getTitle())</label>
```

Comment:
Okay, last comment: please move this into a CSS (SASS for morpheus) class instead of hard-coding opacity